PyRoute/TradeCalculation.py

```diff
@@ -691,6 +696,21 @@ def route_distance(route):
             start = end
         return distance
 
+    def route_cost(self, route):
+        """
+        Given a route, return its total cost via _compensated_ summation
+        """
+        total_weight = 0
```

Comment:
I don't see where this assigns a value to `total_weigh` before returning it. So it always returns 0?

Reply:
Good catch, completely missed that goof.

Reply:
@tjoneslo , this should be fixed in https://github.com/makhidkarun/traveller_pyroute/pull/34/commits/93eb6f06c1738b65f61d1b6c013b35b439f36565 - some wombat had spaced fulling the updated sum back _out_ .